web/src/project-activity.js

```diff
@@ -13,6 +13,8 @@ import { orderResultsByProjectName } from './model/project-actions';
 import './project-activity.css';
 import ModalProgress from './modal-progress';
 
+const SYSTEM_RESET_RECOMMENDATION = 'Run SYSTEM > RESET to ensure engine changes take effect';
```

Comment:
SYSTEM > RESET overwrites a user's current audio levels (including reverb, compressor, and softcut) and favorites. could we change this text to say "restart norns to ensure engine changes take effect"?

it might be way outside of scope, but would it be possible/helpful to have a macro button in this dialogue that just executes `;restart` to the SC and then matron layers? so a user can follow a one-stop action? (@tehn , or would a system "restart" action which could do the same make more sense?)